### PR TITLE
Fixed several commands to be copy-pastable in getting started guide

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/221/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/221/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-221/output/_site/

File: getting-started/installation.markdown [hunk 1]
```diff
@@ -101,23 +112,38 @@ Depending on your operating system and how you installed python, you may be able
...
 There are many ways to install command line tools with `pip`, if you want to do it without `sudo`, and install it in your home directory, and edit the `PATH` variable, or if you want to use virtual environments, you can.
 The commands above are suggestions which should work for most people.
 Importantly, you need the command line tools working after you've installed them:
 
 ```
-$ cfbs --version
+cfbs --version
+```
+
+Just as above, with python, you should see the version number like this:
```
Comment (larsewi) [inline getting-started/installation.markdown:132]:
```suggestion
Just as above – with python – you should see the version number like this:
```

File: getting-started/installation.markdown [hunk 2]
```diff
@@ -164,16 +189,20 @@ If you see `hello` printed, it worked! If not, these are some of the more common
...
 **Note:** If you are not using the vagrant machine with that IP address, remember to replace it with the one you use.
 
 The host is now in a `cf-remote` group called `hub`, so we don't have to type the username and IP, for example:
 
 ```
-$ cf-remote info -H hub
+cf-remote info -H hub
+```
+
+The output shows you the information needed for SSH (username and hostname / IP) as well as some key information about the host, such as architecture and operating system:
```
Comment (larsewi) [inline getting-started/installation.markdown:203]:
```suggestion
The output shows you the information needed for SSH (username and hostname/IP) as well as some key information about the host, such as architecture and operating system:
```

**Review by larsewi — APPROVED** [review]
:1st_place_medal:

**Pull request merged**